bindings/rust/wasmedge-sys/src/vm.rs

```diff
@@ -360,6 +363,25 @@ impl Vm {
         self.run_function_async(func_name, params)
     }
```

Comment:
Please replace the original `run_wasm_from_file_async` with `run_wasm_from_file_async2`, but keep using the original API name `run_wasm_from_file_async`. Thanks!